soh/src/code/z_lifemeter.c

```diff
@@ -162,9 +162,9 @@ void HealthMeter_Init(GlobalContext* globalCtx) {
     sHeartsDDPrim[0][1] = sHeartsDDPrim[1][1] = HeartDDOutline[1];
     sHeartsDDPrim[0][2] = sHeartsDDPrim[1][2] = HeartDDOutline[2];
 
-    sHeartsDDPrim[2][0] = HeartInner[0];
-    sHeartsDDPrim[2][1] = HeartInner[1];
-    sHeartsDDPrim[2][2] = HeartInner[2];
+    // sHeartsDDPrim[2][0] = HeartInner[0];
+    // sHeartsDDPrim[2][1] = HeartInner[1];
+    // sHeartsDDPrim[2][2] = HeartInner[2];
```

Comment:
Why are these removed?

Reply:
I removed these because they are OOB writes (if you look at `sHeartsDDPrim` it's defined as `sHeartsDDPrim[2][3]`). I don't know what the intentions of the person that added these were but I assumed it was a mistake here.